packages/grid/src/GridRenderer.ts

```diff
@@ -1710,7 +1930,20 @@ class GridRenderer {
     context.restore();
   }
 
-  drawSelectedRanges(context, state, viewport = {}) {
+  drawSelectedRanges(
+    context: CanvasRenderingContext2D,
+    state: GridRenderState,
+    viewport: {
+      left?: VisibleIndex;
+      top?: VisibleIndex;
+      right?: VisibleIndex;
+      bottom?: VisibleIndex;
+      minX?: VisibleIndex;
+      minY?: VisibleIndex;
+      maxX?: VisibleIndex;
+      maxY?: VisibleIndex;
```

Comment:
Are these actually indexes? Looks like the default values set are coordinates/pixels

Reply:
They are indeed `Coordinate`, nice catch.